steelscript/appresponse/core/reports.py

```diff
@@ -301,6 +294,9 @@ def __init__(self, source, columns, start=None, end=None, duration=None,
                                      duration=duration, time_range=time_range)
         self._filters = []
         self._data = None
+        self.limit = limit
+        self.topbycolumns = topbycolumns
+        
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008) there should only be one line between methods of a class.